src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub fn yank(string: &str) {
         "freebsd",
         "netbsd",
         "dragonfly",
+        "mac osx",
```

Comment:
The reason "mac osx" was not on this list before is that (as far as I know) it handles clipboard functionality just fine without this hack. This is not a "supported operating systems" list but instead a list of operating systems for which a certain workaround is required. Please remove it from this list.